packages/insomnia-app/app/ui/redux/selectors.ts

```diff
@@ -84,6 +84,11 @@ export const selectActiveSpace = createSelector(
   },
 );
 
+export const selectSpaceSortOrder = createSelector(
+  (state: RootState) => state.global.spaceSortOrder,
+  sortOrder => sortOrder
+);
```

Comment:
This should be the following to match the existing global selectors (see the bottom of this file)

```suggestion
export const selectSpaceSortOrder = createSelector(
  selectGlobal,
  global => global.spaceSortOrder,
);
```